src/components/Button/Button.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './button.css';
+
+export const Button = () => {
+  return (
+    <button className="button" aria-label="Load more" onClick={() => console.log('first')}>
+      Load more
+    </button>
```

Comment:
Add props to this component. Pass `title`, `onClick` and `className` from outside.
Note: you should keep `"button"` className as is, but add possibility to pass extra className from outside.

Use `classnames` package for this purpose